3rdParty/MemPlumber/MemPlumber/CMakeLists.txt

```diff
@@ -1,24 +1,36 @@
-cmake_minimum_required (VERSION 3.0)
-project (memplumber CXX)
+cmake_minimum_required(VERSION 3.0)
+project(memplumber CXX)
 
-add_library (memplumber
-	memplumber.cpp
-	memplumber.h
-)
+add_library(memplumber memplumber.cpp)
 
 # Build options
-option(COLLECT_STATIC_VAR_DATA        "Collect data also on static variable memory allocation" OFF)
+option(COLLECT_STATIC_VAR_DATA
+       "Collect data also on static variable memory allocation" OFF)
+option(DISABLE_BACKTRACE "Disable backtrace" OFF)
 
 if(COLLECT_STATIC_VAR_DATA)
-    add_definitions(-DCOLLECT_STATIC_VAR_DATA)
+  add_definitions(-DCOLLECT_STATIC_VAR_DATA)
 endif()
 
-target_include_directories(memplumber PUBLIC
-	${CMAKE_CURRENT_SOURCE_DIR}
-)
+target_include_directories(memplumber PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+
+# execinfo for backtraces. This is only required on FreeBSD/Alpine, as both Mac
+# OS X and Linux include backtrace functions in libSystem / libc.
+find_package(Backtrace)

```

Comment:
It looks like there is a built-in cmake package `FindBacktrace` that we can use:
https://cmake.org/cmake/help/latest/module/FindBacktrace.html